### ISSUE TITLE
Annoying Perl warnings in recent builds.

### ISSUE THREAD

**Issue by vdukhovni** [openssl/openssl#7634]
In recent builds of 1.1.1, there are many warnings output, all:
```
Use of uninitialized value in pattern match (m//) at ./util/add-depends.pl line 40.
```

**Comment by levitte** [comment]
Hmmm, how fresh is your checkout?  I cherry-picked the fix yesterday: 02d3c6aecc646872af1286144ce8af0693a9f4e3

**Comment by levitte** [comment]
Oops, I see them too.  Darn...

**Comment by levitte** [comment]
Another question: does this happen only when configured `no-shared`?

**Comment by vdukhovni** [comment]
I never use "no-shared".

**Comment by levitte** [comment]
I found the problem, and it wasn't tied to `no-shared`.  PR coming up!

**Comment by levitte** [comment]
See #7635

**Comment by vdukhovni** [comment]
Addressed by PR. Closing.

**Comment by levitte** [comment]
You didn't have to close this manually, github would have done that automatically when i finally pushed 0c594cc, because the commit message contains `Fixes #7634`.  Maybe you would have closed manually anyway, and that's of course fine!  This is just a tip that you don't have to bother if you don't want to.
